src/App.js

```diff
@@ -1,15 +1,17 @@
-import React, { Component } from 'react';
-import DataTable from './DataTable';
-import './App.css';
+import React from "react";
 
-class App extends Component {
-  render() {
-    return (
-      <div className="container mt-3">
-        <DataTable rows={this.props.rows} locale="da" rowsPerPage={5} />
-      </div>
-    );
-  }
-}
+import "./App.css";
+import DataTable from "./DataTable";
+import { userData } from "./data";
+
+// both machines and human love functional component, more readable, less buggy they are
+const App = () => {
+  // it is better to have fixed data table
+  return (
+    <div className="container mt-3">
+      <DataTable rows={userData} locale="da" rowsPerPage={5} />
```

Comment:
Locale:
we should either remove locale or

save user locale in localstorage or getting it from browser settings